community/io/src/main/java/org/neo4j/io/fs/DefaultFileSystemAbstraction.java

```diff
@@ -206,6 +206,12 @@ public long lastModifiedTime( File file )
         return file.lastModified();
     }
 
+    @Override
+    public void deleteFileOrThrow( File file ) throws IOException
```

Comment:
There already exists `boolean deleteFile(File)` method. I'd expect it to use `deleteFileOrThrow()` but it does not and has different implementation (uses `FileUtils` instead of `Files`). This looks quite suspicious. Can we make `deleteFile()` use `deleteFileOrThrow()` and return `true` on success and `false` on exception?